src/sas/qtgui/MainWindow/MainWindow.py

```diff
@@ -1,11 +1,10 @@
 # UNLESS EXEPTIONALLY REQUIRED TRY TO AVOID IMPORTING ANY MODULES HERE
 # ESPECIALLY ANYTHING IN SAS, SASMODELS NAMESPACE
-import logging
 import os
 import sys
+import logging
 
-from sas import config
```

Comment:
there are still uses of `config` in this file; lines 77-80 for the `QT_*` variables.

Reply:
Merge error. Fixing.

I might move the QT_* stuff to cli.py. Are we going to have scripts that invoke Qt that need these variables set?